lib/taskana-data/src/main/java/pro/taskana/sampledata/SampleDataGenerator.java

```diff
@@ -69,11 +69,15 @@ public void generateSampleData() {
           }
           executeAndCacheScripts(scripts, cacheKey);
         });
-    LOGGER.debug("exit from generateSampleData()");
+    if (LOGGER.isDebugEnabled()) {
```

Comment:
We do not need the guarded logging here, as this is the first call in the logger.